include/bluetooth/mesh/sensor.h

```diff
@@ -501,7 +501,7 @@ const char *bt_mesh_sensor_ch_str_real(const struct sensor_value *ch);
  *
  *  @param[in] ch Sensor channel to represent.
  */
-#define bt_mesh_sensor_ch_str(ch) log_strdup(bt_mesh_sensor_ch_str_real(ch))
+#define bt_mesh_sensor_ch_str(ch) bt_mesh_sensor_ch_str_real(ch)
```

Comment:
This redefinition is useless now and can be removed. Just rename function `bt_mesh_sensor_ch_str_real` to `bt_mesh_sensor_ch_str` to make proper changes.

Reply:
Resolved